.coveragerc

```diff
@@ -12,7 +12,7 @@ omit =
   *wsgi.py
   *migrations/*
   *test/*
-  */tests_*.py
+  */test_*.py
```

Comment:
Would `*/test*.py` make more sense to cover all the bases? I'm not sure how consistently we want to name the tests.

Reply:
done.